R/provider-openrouter.R

```diff
@@ -116,19 +116,18 @@ check_openrouter_error <- function(error, call = caller_env()) {
 }
 
 method(chat_resp_stream, ProviderOpenRouter) <- function(provider, resp) {
-  repeat
-    ({
```

Comment:
Idk what was going on with the `({` here, but I remove the `(` and it reflowed properly